### ISSUE TITLE
Bot not registering game is running

### ISSUE THREAD

**Issue by IanKoss** [Masuzu/ZooeyBot#1303]
**Describe the bug**
Hello again, I looked at your guide and setup instructions and I did all that it said to do. But when I try to run it it does not say GBF is running even though it clearly is on the tab and the picture shows it as well. I do not know what I did wrong
![Capture](https://user-images.githubusercontent.com/25437967/79811047-ae165200-8342-11ea-9b15-0d23e57590dc.PNG)

 for it not to register that it is active.

**Screenshots**

**Chat directly with us**
https://discordapp.com/invite/JaNR87n

**Comment by ramblinguy** [comment]
It's been a while since I saw the error, but did you set your "language='en'" in the ini file? Not sure if that could cause this error

**Comment by IanKoss** [comment]
I do not know, I am unable to find what you mean by the ini file. I am somewhat new to this so sorry if I do not know some of the basic meanings or phrases.

**Comment by ramblinguy** [comment]
when you download and unzip the bot, you should see a "ZooeyBot.ini". If you open it in an editor (I use Notepad++, but I think Notepad should work. Just not Word), you will see at the top:
![image](https://user-images.githubusercontent.com/23442109/79818559-1f133500-8356-11ea-8c07-bff6e9024bc0.png)

Make sure that it's set to en and not jp

**Comment by IanKoss** [comment]
I do not see it even though i installed and extracted it. Ill try re downloading it

**Comment by IanKoss** [comment]
![Capture](https://user-images.githubusercontent.com/25437967/79820080-65b65e80-8359-11ea-8e47-353840c4bf01.PNG)
This is all that is in the folder when I download it. Is there a different link i get it from than whats on the website?

**Comment by ramblinguy** [comment]
Oh you have "hide extensions for known file types" turned on. The "ZooeyBot" under the application that has "Configuration Settings" next to it is the "ini" file. You're going to be editing that a lot for your quests and stuff. I would highly suggest reading the wiki and learning how to use Lua scripts, or else the bot won't be that useful.
https://github.com/Masuzu/ZooeyBot/wiki

**Comment by IanKoss** [comment]
Thank you so much! I got it working to an extent, and i starting reading through the guides. All I want to do it make it go in auto mode auto replenishing my AP while doing the Elemental Treasure quests. I found the option to do what I need and configured it. But I have one last pickle. I follow the instructions setting the things I need to to false so i can do the automated special quests but it doesn't take over. 
![Capture](https://user-images.githubusercontent.com/25437967/79822442-268b0c00-835f-11ea-885e-1a9316eee8d6.PNG)
I have my dev thing up and it gets to "stage 2" or something and I don't see anything on how to get pass this so it can take control.
